stdlib/_markupbase.pyi

```diff
@@ -5,9 +5,9 @@ class ParserBase:
     def reset(self) -> None: ...
     def getpos(self) -> tuple[int, int]: ...
     def unknown_decl(self, data: str) -> None: ...
-    def parse_comment(self, i: int, report: int = 1) -> int: ...  # undocumented
+    def parse_comment(self, i: int, report: bool | int = 1) -> int: ...  # undocumented
     def parse_declaration(self, i: int) -> int: ...  # undocumented
-    def parse_marked_section(self, i: int, report: int = 1) -> int: ...  # undocumented
+    def parse_marked_section(self, i: int, report: bool | int = 1) -> int: ...  # undocumented
```

Comment:
(GitHub is being weird again and not showing the full diff, even after asking it to.)

Same here.